technical-report/sections/03-termsdef.adoc

```diff
@@ -0,0 +1,128 @@
+
+[[terms]]
+== Terms and definitions
+
+
+=== translation type
+
+methods used when translating between ODF and OOXML documents
+
+[NOTE]
+====
+This Technical Report distinguishes four translation types:
+* one way ODF to OOXML translation;
+* one way OOXML to ODF translation;
+* round trip ODF to OOXML to ODF translation;
+* round trip OOXML to ODF to OOXML translation.
+====
+
+=== translation fidelity
+
+quality of a translation process between the ODF and OOXML document formats
+NOTE: Translation fidelity depends on document properties.
+
+NOTE: Translation fidelity cannot be measured in an absolute manner; it depends on the intentions of the document's authors.
+
+
+=== document type
```

Comment:
Please remove the terms that have less illustrative value for the author.

Reply:
Done